rust-runtime/aws-smithy-runtime-api/src/client/runtime_components.rs

```diff
@@ -634,6 +634,14 @@ impl RuntimeComponentsBuilder {
         self
     }
 
+    /// Returns [`SharedIdentityResolver`] configured in the builder for a given `scheme_id`.
+    pub fn identity_resolver(&self, scheme_id: &AuthSchemeId) -> Option<SharedIdentityResolver> {
```

Comment:
Usage of this depends on order right? S3 express is using this and implicitly requires that identity resolvers already be set on runtime components, how is that enforced?

Reply:
Yes, `S3ExpressRuntimePlugin` kicks in after `ServiceRuntimePlugin`. Added comment in 073f673.